aspnetcore/performance/caching/response.md

```diff
@@ -20,7 +20,7 @@ uid: performance/caching/response
 
 ## What is Response Caching
 
-*Response caching* refers to specifying cache-related headers on HTTP responses made by ASP.NET Core MVC actions. These headers specify how you want client and intermediate (proxy) machines to cache responses to certain requests (if at all). This can reduce the number of requests a client or proxy makes to the web server, since future requests for the same action may be served from the client or proxy's cache. In this case, the request is never made to the web server.
+*Response caching* refers to specifying cache-related headers on HTTP responses made by ASP.NET Core MVC actions. These headers specify how you want client, intermediate (proxy) machines and middleware to cache responses to certain requests (if at all). This can reduce the number of requests a client or proxy makes to the web server or reduce the amount of work the web server performs to generate the response, since future requests for the same action may be served from the cache of the client, proxy or server. In this case, either the request is never made to the web server or is served by the response caching middleware on the server.
 
 ![image](response/_static/proxy-and-cache.png)
```

Comment:
this graphic might need to be updated to include the caching done by the middleware on the server?